delivery_postlogistics/models/stock_move.py

```diff
@@ -10,13 +10,14 @@ class StockMove(models.Model):
     def _get_new_picking_values(self):
         vals = super(StockMove, self)._get_new_picking_values()
 
-        order_commitment_date = (
-            self.sale_line_id and self.sale_line_id.order_id.commitment_date
-        )
-
-        if order_commitment_date:
+        order_commitment_dates = [
```

Comment:
We have to consider the `shipping_policy`.
If `all at once` use `max()`, if `asap` use `min()`

Also, commitment date might be False in some cases. This isn't a mandatory field
```python3
>>> max([False, datetime.now()])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'datetime.datetime' and 'bool'
```